procmon/processmonitor.cpp

```diff
@@ -809,6 +810,24 @@ void ProcessMonitor::processMessage(messageqcpp::ByteStream msg, messageqcpp::IO
                     if ( processName == "cpimport" )
                     {
                         system("pkill -sighup cpimport");
+                        for (int i=0; i < 10; i++)
+                        {
+                            //get pid
+                            char buf[512];
+                            FILE *cmd_pipe = popen("pidof -s cpimport", "r");
+
+                            fgets(buf, 512, cmd_pipe);
+                            pid_t pid = strtoul(buf, NULL, 10);
+
+                            pclose( cmd_pipe );
+
+                            if (pid)
+                                sleep(2);
+                            else
+                                break;
+                        }
```

Comment:
Looks like the purpose of this loop is to wait up to 20s for cpimport to go away before killing cpimport.bin on the local machine, is that right?